src/test/unit/flight_imu_unittest.cc

```diff
@@ -68,131 +66,6 @@ extern "C" {
 #include "unittest_macros.h"
 #include "gtest/gtest.h"
 
-const float sqrt2over2 = sqrt(2) / 2.0f;
```

Comment:
Where did these tests go?

Reply:
@mikeller 
these were comparisons 
between rMat and quaternion calculus with 1e-6 precision.
They are no longer needed.